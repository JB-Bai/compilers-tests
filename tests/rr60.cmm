struct{
	int OiyqkChF[5200] = ! ddkHDOd;
}
;
struct l3De__y BseX7YD(int tlXpouW_[61506], struct LYJQRQ R6QsydgPf[0xBD92][0x0], float orgtpmm, int jhV_t[607]){
	struct Tk2DVEK SP01lHeg = a8XFDqQBB(), O3Ly01e, m0D7_6, jFM6u0[0x225] = !(jnkMfChf4) = 0.46[RycKaZT] && RjjeUeQ7h;
	float sa267[0x6F][0x9a], F7w9;
	struct{
		int PdMxvR = 649782, QWXmJPZa = 23856.yNvgMlWIi;
	}
	HOzJxAFNr[932], OKyTit, H6ZWwBPe;
	int kKxvuV7[927] = 6655[07411];
	int sDGgwTnsM[04307] = 306.91 && ! 7.696 == 8.877 =(81.089).T7xSUU < ! - 0xAd5 || 19.767 / 0x17b9 / 1, idrHB7O, dqvOua = 6 <= zUho0s9 && g8MYdBsZGd() = - 926.766E5 || DBlk3LdLr();
	0207 = - DFOVl2DdpbL && Q5t5Wl0P682I.s9eHTsjH;
}
struct VQqkzs0hE{
}
;

