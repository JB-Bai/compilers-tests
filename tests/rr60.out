Program (1)
  ExtDefList (1)
    ExtDef (1)
      Specifier (1)
        StructSpecifier (1)
          STRUCT
          LC
          DefList (2)
            Def (2)
              Specifier (2)
                TYPE: int
              DecList (2)
                Dec (2)
                  VarDec (2)
                    VarDec (2)
                      ID: OiyqkChF
                    LB
                    INT: 5200
                    RB
                  ASSIGNOP
                  Exp (2)
                    NOT
                    Exp (2)
                      ID: ddkHDOd
              SEMI
          RC
      SEMI
    ExtDefList (5)
      ExtDef (5)
        Specifier (5)
          StructSpecifier (5)
            STRUCT
            Tag (5)
              ID: l3De__y
        FunDec (5)
          ID: BseX7YD
          LP
          VarList (5)
            ParamDec (5)
              Specifier (5)
                TYPE: int
              VarDec (5)
                VarDec (5)
                  ID: tlXpouW_
                LB
                INT: 61506
                RB
            COMMA
            VarList (5)
              ParamDec (5)
                Specifier (5)
                  StructSpecifier (5)
                    STRUCT
                    Tag (5)
                      ID: LYJQRQ
                VarDec (5)
                  VarDec (5)
                    VarDec (5)
                      ID: R6QsydgPf
                    LB
                    INT: 48530
                    RB
                  LB
                  INT: 0
                  RB
              COMMA
              VarList (5)
                ParamDec (5)
                  Specifier (5)
                    TYPE: float
                  VarDec (5)
                    ID: orgtpmm
                COMMA
                VarList (5)
                  ParamDec (5)
                    Specifier (5)
                      TYPE: int
                    VarDec (5)
                      VarDec (5)
                        ID: jhV_t
                      LB
                      INT: 607
                      RB
          RP
        CompSt (5)
          LC
          DefList (6)
            Def (6)
              Specifier (6)
                StructSpecifier (6)
                  STRUCT
                  Tag (6)
                    ID: Tk2DVEK
              DecList (6)
                Dec (6)
                  VarDec (6)
                    ID: SP01lHeg
                  ASSIGNOP
                  Exp (6)
                    ID: a8XFDqQBB
                    LP
                    RP
                COMMA
                DecList (6)
                  Dec (6)
                    VarDec (6)
                      ID: O3Ly01e
                  COMMA
                  DecList (6)
                    Dec (6)
                      VarDec (6)
                        ID: m0D7_6
                    COMMA
                    DecList (6)
                      Dec (6)
                        VarDec (6)
                          VarDec (6)
                            ID: jFM6u0
                          LB
                          INT: 549
                          RB
                        ASSIGNOP
                        Exp (6)
                          Exp (6)
                            NOT
                            Exp (6)
                              LP
                              Exp (6)
                                ID: jnkMfChf4
                              RP
                          ASSIGNOP
                          Exp (6)
                            Exp (6)
                              Exp (6)
                                FLOAT: 0.460000
                              LB
                              Exp (6)
                                ID: RycKaZT
                              RB
                            AND
                            Exp (6)
                              ID: RjjeUeQ7h
              SEMI
            DefList (7)
              Def (7)
                Specifier (7)
                  TYPE: float
                DecList (7)
                  Dec (7)
                    VarDec (7)
                      VarDec (7)
                        VarDec (7)
                          ID: sa267
                        LB
                        INT: 111
                        RB
                      LB
                      INT: 154
                      RB
                  COMMA
                  DecList (7)
                    Dec (7)
                      VarDec (7)
                        ID: F7w9
                SEMI
              DefList (8)
                Def (8)
                  Specifier (8)
                    StructSpecifier (8)
                      STRUCT
                      LC
                      DefList (9)
                        Def (9)
                          Specifier (9)
                            TYPE: int
                          DecList (9)
                            Dec (9)
                              VarDec (9)
                                ID: PdMxvR
                              ASSIGNOP
                              Exp (9)
                                INT: 649782
                            COMMA
                            DecList (9)
                              Dec (9)
                                VarDec (9)
                                  ID: QWXmJPZa
                                ASSIGNOP
                                Exp (9)
                                  Exp (9)
                                    INT: 23856
                                  DOT
                                  ID: yNvgMlWIi
                          SEMI
                      RC
                  DecList (11)
                    Dec (11)
                      VarDec (11)
                        VarDec (11)
                          ID: HOzJxAFNr
                        LB
                        INT: 932
                        RB
                    COMMA
                    DecList (11)
                      Dec (11)
                        VarDec (11)
                          ID: OKyTit
                      COMMA
                      DecList (11)
                        Dec (11)
                          VarDec (11)
                            ID: H6ZWwBPe
                  SEMI
                DefList (12)
                  Def (12)
                    Specifier (12)
                      TYPE: int
                    DecList (12)
                      Dec (12)
                        VarDec (12)
                          VarDec (12)
                            ID: kKxvuV7
                          LB
                          INT: 927
                          RB
                        ASSIGNOP
                        Exp (12)
                          Exp (12)
                            INT: 6655
                          LB
                          Exp (12)
                            INT: 3849
                          RB
                    SEMI
                  DefList (13)
                    Def (13)
                      Specifier (13)
                        TYPE: int
                      DecList (13)
                        Dec (13)
                          VarDec (13)
                            VarDec (13)
                              ID: sDGgwTnsM
                            LB
                            INT: 2247
                            RB
                          ASSIGNOP
                          Exp (13)
                            Exp (13)
                              Exp (13)
                                FLOAT: 306.910004
                              AND
                              Exp (13)
                                Exp (13)
                                  NOT
                                  Exp (13)
                                    FLOAT: 7.696000
                                RELOP
                                Exp (13)
                                  FLOAT: 8.877000
                            ASSIGNOP
                            Exp (13)
                              Exp (13)
                                Exp (13)
                                  Exp (13)
                                    LP
                                    Exp (13)
                                      FLOAT: 81.088997
                                    RP
                                  DOT
                                  ID: T7xSUU
                                RELOP
                                Exp (13)
                                  NOT
                                  Exp (13)
                                    MINUS
                                    Exp (13)
                                      INT: 2773
                              OR
                              Exp (13)
                                Exp (13)
                                  Exp (13)
                                    FLOAT: 19.767000
                                  DIV
                                  Exp (13)
                                    INT: 6073
                                DIV
                                Exp (13)
                                  INT: 1
                        COMMA
                        DecList (13)
                          Dec (13)
                            VarDec (13)
                              ID: idrHB7O
                          COMMA
                          DecList (13)
                            Dec (13)
                              VarDec (13)
                                ID: dqvOua
                              ASSIGNOP
                              Exp (13)
                                Exp (13)
                                  Exp (13)
                                    Exp (13)
                                      INT: 6
                                    RELOP
                                    Exp (13)
                                      ID: zUho0s9
                                  AND
                                  Exp (13)
                                    ID: g8MYdBsZGd
                                    LP
                                    RP
                                ASSIGNOP
                                Exp (13)
                                  Exp (13)
                                    MINUS
                                    Exp (13)
                                      FLOAT: 92676600.000000
                                  OR
                                  Exp (13)
                                    ID: DBlk3LdLr
                                    LP
                                    RP
                      SEMI
          StmtList (14)
            Stmt (14)
              Exp (14)
                Exp (14)
                  INT: 135
                ASSIGNOP
                Exp (14)
                  Exp (14)
                    MINUS
                    Exp (14)
                      ID: DFOVl2DdpbL
                  AND
                  Exp (14)
                    Exp (14)
                      ID: Q5t5Wl0P682I
                    DOT
                    ID: s9eHTsjH
              SEMI
          RC
      ExtDefList (16)
        ExtDef (16)
          Specifier (16)
            StructSpecifier (16)
              STRUCT
              OptTag (16)
                ID: VQqkzs0hE
              LC
              RC
          SEMI
